docs/sql-guide/statements/statement-columns-show.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SHOW COLUMNS
+layout: default
+parent: Statements
+grand_parent: SQL guide
+---
+
+# SHOW COLUMNS statement
+
+## BNF diagram
+
+![expr](/assets/images/sql-guide/show_columns.svg)
+
+## DDL Syntax
+
+```sql
+SHOW COLUMNS FROM table_name;
+```
+
+## Arguments
+
+| Argument | Description |
+|---|---|
+| table_name | name of existing table in database |
+
+## Returns
+
+| Column name | Data type | Description | Further information |
+|---|---|---|
+| `name` | string | column name |  |
+| `type` | string | column data type | [Data types](/docs/sql-guide/data-types/data-types-home) |
+| `internal_type` | string | column data type |  |
+| `created_at` | timestamp | timestamp | [TIMESTAMP() data type](/docs/sql-guide/data-types/data-type-timestamp) |
+| `keys` | bool | Keys on the column |  |
+| `cache_type` | string |  |  |
+| `cache_size` | int |  |  |
+| `scale` | int | Scale value for `DECIMAL()` column | [DECIMAL() data type](/docs/sql-guide/data-types/data-type-decimal) |
+| `min` | int | Minimum value for `INT()` column | [INT() data type](/docs/sql-guide/data-types/data-type-int) |
+| `max` | int | Maximum value for `INT()` column | [INT() data type](/docs/sql-guide/data-types/data-type-int) |
+| `timeunit` | string | TIMEUNIT for `TIMESTAMP()` column | [TIMESTAMP() data type](/docs/sql-guide/data-types/data-type-timestamp) |
+| `epoch` | int | Epoch for `TIMESTAMP()` column | [TIMESTAMP() data type](/docs/sql-guide/data-types/data-type-timestamp) |
+| `time_quantum` | string | Applies to `IDSET()` and `STRINGSET()` columns | [IDSET() data type](/docs/sql-guide/data-types/data-type-idset)<br/>[STRINGSET() data type](/docs/sql-guide/data-types/data-type-stringset) |
+| `ttl` | string | Applies to `IDSET()` and `STRINGSET()` columns | [IDSET() data type](/docs/sql-guide/data-types/data-type-idset)<br/>[STRINGSET() data type](/docs/sql-guide/data-types/data-type-stringset) |
+
+## Examples
+
+### SHOW COLUMNS on sample database `skills` table
+
+```
+SHOW COLUMNS FROM skills;
+```
+
+Returns
+
+|_id | name | type | internal_type | created_at | keys | cache_type | cache_size | scale | min | max | timeunit | epoch | timequantum | ttl |
```

Comment:
did you pull these results from cloud? seems like type and internal type never differ which I didn't expect but could be now...

Reply:
Yeah this was pulled from cloud with the sample database setup

Reply:
And I've tested it against my simple tables and cols are the same